plugins/woocommerce/includes/admin/settings/class-wc-settings-page.php

```diff
@@ -84,6 +84,42 @@ public function add_settings_page( $pages ) {
 			return $pages;
 		}
 
+		/**
+		 * Get page settings data to populate the settings editor.
+		 *
+		 * @param array $pages The settings array where we'll add data.
+		 *
+		 * @return array
+		 */
+		public function get_settings_page_data( $pages ) {
```

Comment:
Any opportunity to add some tests here?

Reply:
Oh yes, added in https://github.com/woocommerce/woocommerce/pull/52472/commits/8d531319d31aa64d0de529eedf77b605c611c7d3